flowetl/Dockerfile

```diff
@@ -27,6 +27,10 @@ RUN USER=airflow && \
 RUN mv /entrypoint.sh /defaultentrypoint.sh
 COPY ./entrypoint.sh /entrypoint.sh
 
+WORKDIR /libs
+COPY ./etl /libs/etl
```

Comment:
Doesn't really matter which directory we use since it's inside the container, but for cosmetic reasons how about using `/opt/etl` instead? (Conforms a little better to traditional Unix folder structure.)